docs/userGuide/syntax/boxes.mbdf

```diff
@@ -78,6 +78,60 @@
 </span>
 </include>
 
+**Markbind also supports a light color scheme for boxes**
+
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+```html
+<box type="info" light>
+    info light
+</box>
+<box type="warning" light>
+    warning light
+</box>
+<box type="success" light>
+    success light
+</box>
+<box type="important" light>
+    important light
+</box>
+<box type="wrong" light>
+    wrong light
+</box>
+<box type="tip" light>
+    tip light
+</box>
+<box type="definition" light>
+    definition light
+</box>
+```
+</span>
+<span id="output">
+
+<box type="info" light>
+    info light
+</box>
+<box type="warning" light>
+    warning light
+</box>
+<box type="success" light>
+    success light
+</box>
+<box type="important" light>
+    important light
+</box>
+<box type="wrong" light>
+    wrong light
+</box>
+<box type="tip" light>
+    tip light
+</box>
+<box type="definition" light>
+    definition light
+</box>
+</span>
+</include>
 
 **You can customize the TipBox's appearance.**
```

Comment:
As long as we are touching this code, can we also change this `TipBox` -> `box`?